data/json/professions.json

```diff
@@ -5158,5 +5158,81 @@
                 "panties"
             ]
         }
+    },
+        {
+        "type": "profession",
+        "ident": "bio_sci",
+        "name": "Bionics Scientist",
+        "description": "Before all this you studied bionics. You tried your experiments on animals and yourself.", 
+        "traits": [
+            "ANIMALDISCORD",
+            "PSYCHOPATH"
```

Comment:
Idea: make this "Bionics Mad Scientist" and update the description to give it a little more crazed flavor. 

Psychopath trait and the scientist tried their experiments on *themselves*? Definitely insane. Perhaps add a single failed bionic? Surely not _all_ those experiments went well... 😈

Reply:
1 change is done, let me check the other classes in how I'd add a *broken" bionic.